teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>I suggest to move (or remove) this sentence 'In this tutorial, you learn how to generate a radar plot using the Python plotting library&nbsp;<code>matplotlib</code>.&nbsp;'</p><p>Because here it's more about the theoretical aspect.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Agreed.</p>